radio/src/gui/common/stdlcd/radio_sdmanager.cpp

```diff
@@ -105,13 +105,14 @@ void onSdManagerMenu(const char * result)
     memcpy(reusableBuffer.sdmanager.originalName, line, sizeof(reusableBuffer.sdmanager.originalName));
     char * ext = getFileExtension(line, SD_SCREEN_FILE_LENGTH+1);
     if (ext) {
+      size_t len = strlen(ext);
```

Comment:
Another thought here...  getFileExtension() already does a strlen(filename) on the file passed to it.  Doing it again in any way is redundant... we could add an optional param to getFileExtension() into which to place the full length of the filename with ext.  Seems obvious in retrospect...  perhaps this can save some time in other places as well.

Although my points A and B above still apply.